packages/authentication/cardstack/middleware.js

```diff
@@ -143,6 +128,24 @@ class Authentication {
       user = await this._processExternalUser(result, source);
     }
 
+    if (result && result.meta && result.meta.partialSession) {
+      if (!user) {
+        // top-level meta is not passed through (it was for
+        // communicating from plugin to us). Plugins could use
+        // resource-level metadata instead if they want to.
+        delete result.meta;
+        user = result;
+      }
```

Comment:
I don't think we really need this whole block anymore.

Reply:
Ok